web_src/js/components/DiffFileTree.vue

```diff
@@ -34,6 +34,14 @@ export default {
     };
   },
   computed: {
+    hasMore: {
+      get() {
+        return this.isIncomplete;
+      },
+      set(newValue) {
+        this.isIncomplete = newValue;
+      }
+    },
```

Comment:
What are you doing here .....

Why a "computed" property should be "set"?

Reply:
The key point of the problem here, it's not about "updating isIncomplete in computed property by tricks", the root problem is:

When the `pageData.diffFileInfo` changes, it needs to "notify" the Vue component to update its data/properties.

The `computed property`  doesn't need to update anything, it should just provide the computed (read-only) result.

To make the Vue component could be notified by changed `pageData.diffFileInfo`, either:

1. Send an event to the Vue component after `pageData.diffFileInfo` is re-loaded, then Vue component re-read the `pageData.diffFileInfo` to its internal reactive data/properties
2. Use "store" for  `pageData.diffFileInfo`, then any change on  `pageData.diffFileInfo` will trigger the Vue's reactive system to sync the values.